kotest-assertions/kotest-assertions-shared/src/commonMain/kotlin/io/kotest/matchers/MatcherResultWithError.kt

```diff
@@ -1,6 +1,6 @@
 package io.kotest.matchers
 
-internal interface MatcherResultWithError : MatcherResult {
+interface MatcherResultWithError : MatcherResult {
```

Comment:
```suggestion
@KotestInternal
interface MatcherResultWithError : MatcherResult {
```

`@KotestInternal` should be used to flag API:s that are exposed to make them available across modules within Kotest, but which shouldn't be used by Kotest users.

Reply:
Thank, change applied.